### PR TITLE
Add support to load views directly from any registered Tipoff package during testing 

### PR DESCRIPTION
**Pull request by pdbreen**
Also includes a `withViews(...)` helper to add any other paths to the search.

### CODE REVIEW

**Comment by pdbreen** [comment]
I flagged don't merge for now -  I'm pretty sure there will be an issue trying to get views loaded when they exist in multiple packages (shared base component).  I want to make sure there is a solution for that as well before merge.

**Comment by pdbreen** [comment]
This one is good to go.

**Review by drewroberts — APPROVED** [review]
Thank you! 🎷

**Pull request merged**
